composer.json

```diff
@@ -17,7 +17,7 @@
         "php": ">=5.3.0",
         "silex/silex": ">=1.0,<2.0",
         "twig/twig": ">=1.8,<2.0-dev",
-        "symfony/twig-bridge": "~2.3"
+        "symfony/twig-bridge": "~2.3|^3.0"
```

Comment:
Looking at the [docs from composer](https://getcomposer.org/doc/articles/versions.md#range) this should be a double pipe or?